### PR TITLE
Save the native "value" property setter of HTMLInputElement.prototype (close #1185)

### LINKED ISSUE #1185: Save the native "value" property setter of HTMLInputElement.prototype

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @AlexanderMoskovkin @miherlosev

### CODE REVIEW

File: src/client/sandbox/native-methods.js [hunk 1]
```diff
@@ -177,6 +177,9 @@ class NativeMethods {
         if (win.DOMParser)
             this.DOMParserParseFromString = win.DOMParser.prototype.parseFromString;
 
+        // Setters
+        this.inputValueSetter = win.Object.getOwnPropertyDescriptor(win.HTMLInputElement.prototype, 'value').set;
```
Comment (AlexanderMoskovkin) [inline src/client/sandbox/native-methods.js]: I suppose the same is actual for textareas

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

File: test/client/fixtures/sandbox/native-methods-test.js [hunk 2]
```diff
@@ -7,3 +7,33 @@ if (nativeMethods.performanceNow) {
         ok(!isNaN(parseFloat(now)));
     });
 }
+
+if (nativeMethods.inputValueSetter) {
+    test("set element's value with native setter", function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/native-methods-test.js]: `element's prototype value setters`

**Comment by miherlosev** [comment]
@churkin review too

**Pull request merged**